From ad110e5ff36de6096e1b9d7e0fe125326f45ed7d Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 19 Jul 2024 13:18:47 +0100
Subject: [PATCH 1183/1215] arm64: dts: Give cm5l its own model name

The bootloader patches the DT with the correct model string, but it
is better not to rely on that by setting it from the start.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5l.dtsi | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5l.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5l.dtsi
@@ -4,6 +4,8 @@
 #include "bcm2712-rpi-cm5.dtsi"
 
 / {
+	model = "Raspberry Pi Compute Module 5 Lite";
+
 	__overrides__ {
 		i2c_csi_dsi = <&i2c_csi_dsi>, "status";
 	};
